src/SayMore/UI/LowLevelControls/Slider.cs

```diff
@@ -170,8 +173,19 @@ public bool SetValueWithoutEvent(float value)
 		{
 			value = (float)Math.Round(value, 4);
 
-			if (value < 0 || value > Maximum)
-				throw new ArgumentOutOfRangeException("value");
+			if (value < 0)
+			{
+				var msg = LocalizationManager.GetString("CommonToMultipleViews.Slider.ValueIsNegative", "Attempted to set slider to a negative value: {0}.");
```

Comment:
Since the user won't know what a "slider" is, could we possibly figure out a way to provide a more user-friendly name? One possibility might be to use "the {0} slider", where param 0 is **nameof(this)**, but depending on how the sliders are named, that might not be any better. If we can come up with a workable solution, do the same for the other message below.

Reply:
Would it be better to just log the warning and not bother the user?